application/console/config/main.php

```diff
@@ -1,4 +1,13 @@
 <?php
+
+
+use Sil\PhpEnv\Env;
+
+/* For pushing data to Google Analytics */
+$gaTrackingId = Env::get('GA_TRACKING_ID', null); // 'UA-12345678-12'
+$gaClientId = Env::get('GA_CLIENT_ID', null); // 'IDP_ID_BROKER_LOCALHOST'
```

Comment:
The example values in your comments may be worth moving to the `local.env.dist` next to each of these optional env. vars.